corehq/apps/importer/tasks.py

```diff
@@ -78,6 +80,14 @@ def _submit_caseblocks(caseblocks):
                     error=ImportErrors.ImportErrorMessage,
                     row_number=caseblocks[0]._id
                 )
+            else:
+                properties = set().union(*map(lambda c: set(c.dynamic_case_properties().keys()), cases))
+                added_inferred_export_properties.send(
+                    'CaseImporter',
+                    domain=domain,
+                    case_type=case_type,
+                    properties=properties,
+                )
```

Comment:
i don't think we do. the merge will handle the duplicates and i figured it'd add unnecessary complexity to try and filter out the ones that are already in the schema